public/app/features/transformers/docs/content.ts

```diff
@@ -68,7 +68,7 @@ export const transformationDocsContent: TransformationDocsContentType = {
   - **Alias** - (Optional) Enter the name of your new field. If you leave this blank, then the field will be named to match the calculation.
   - **Replace all fields** - (Optional) Select this option if you want to hide all other fields and display only your calculated field in the visualization.
   
-  > **Note:** **Cumulative functions** and **Window functions** modes are currently in private preview. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
+  > **Note:** **Cumulative functions** and **Window functions** modes are currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the addFieldFromCalculationStatFunctions feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```

Comment:
Nit: Should we put the name of the feature flags between quotes?

Reply:
Or back ticks?